src/main/java/com/musala/atmosphere/agent/devicewrapper/AbstractWrapDevice.java

```diff
@@ -165,6 +171,8 @@
 
     private WebElementManager webElementManager;
 
+    private LogcatBuffer<String, Pair<Integer, String>> buffer = new LogcatBuffer<String, Pair<Integer, String>>();
```

Comment:
Initialize the buffer in the constructor. Use more specific name, for example: logcatBuffer.